### ISSUE TITLE
Android port

### ISSUE THREAD

**Issue by krzys-h** [blupi-games/planetblupi#12]
I think it would be great to make an Android port of Planet Blupi. It should be pretty easy to do given the fact that the game uses SDL2 now, which has native support for android. The UI is designed in a way that makes it quite comfortable to use on a touchscreen (almost like they knew 20 years ago!), pretty much the only thing that needs some improvement is the screen scrolling.

I already managed to make a successful build with (almost) no code changes, although my build process is so messy that it probably needs to be redone properly from scratch.
https://youtu.be/PQPCvH_7Cec

**Comment by Skywalker13** [comment]
Yes, that's fun, and for iOS too. I looked a bit for mobile devices too but it was not my priority. If you want to continue in this way, I will be happy to help you. The difficulty is to keep something working (build) on all platforms all the time.

**Comment by Skywalker13** [comment]
@krzys-h hi, do you have something to submit for the Android port? Otherwise I will try to do this work.

**Comment by krzys-h** [comment]
@Skywalker13 I started trying to prepare a cleaner build system for this, but I didn't have time to finish it yet. If I don't finish it soon I'll publish what I have so you can finish it yourself if you want.

**Comment by Skywalker13** [comment]
OK, I wait :-)
